doc/index.rst

```diff
@@ -4,6 +4,13 @@ Arbor
 .. image:: https://travis-ci.org/arbor-sim/arbor.svg?branch=master
     :target: https://travis-ci.org/arbor-sim/arbor
 
+Welcome to the documentation for Arbor, the multi-compartment neural network simulation library.
+
+:ref:`Get Arbor<in_install>`, :ref:`get started<gs_other_examples>` or read on!
```

Comment:
I think the general tone of the docs is quite serious. How about something like, `You can find out how to get Arbor; get started quickly with our tutorials; or continue reading to learn more about Arbor.`

Reply:
Maybe a bit too serious?